src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/GeneralPartnerDataDto.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Size;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.LimitedPartnerType;
+
+import java.time.LocalDate;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+public class GeneralPartnerDataDto {
```

Comment:
Will need a boolean field for "not_disqualified_statement_checked"

Reply:
Still need another field for the 'Does the general partner have a previous name?' question.

Reply:
I don't think we need to store this.

Reply:
I'm fine with that approach @dlloyd1-ch . Will resolve.